.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,208 @@
+
+
+  build_test_deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 5
+
+      - name: "Custom environment variables"
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:50: Double quote to prevent globbing and word splitting

(shellcheck)